decidim-admin/app/assets/stylesheets/decidim/admin/modules/_secondary-nav.scss

```diff
@@ -51,7 +51,8 @@ $process-title-height: 3rem;
 }
 
 .secondary-nav__title,
-.process-title-content{
+.process-title-content,
+.scope-title-content{
```

Comment:
Ok, there are several changes on scopes section that should be reflected on that repo, and also minor changes on participatory processes and other engines. So, the first question is: the synchronization is done by hand? If so, I will fork both repos and submit a PR with the proposed changes and comment there the problems that I found.